manifests/g/Giorgiotani/Peazip/8.4.0/GiorgioTani.PeaZip.installer.yaml

```
@@ -0,0 +1,42 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: GiorgioTani.PeaZip
```

This seems to be a issue on your end?
```suggestion
PackageIdentifier: Giorgiotani.Peazip
```